app/controllers/users/sessions_controller.rb

```diff
@@ -12,6 +12,13 @@ def create
     sign_in(resource_name, resource)
     resource.save # recreates authentication_token after sign in
 
+    graphql_access_token = Doorkeeper::AccessToken.create(
+      application: resource.oauth_applications.first,
```

Comment:
welcher account ist hier first? is das egal? wird irgendeiner verwendet, um ein token zu erstellen? oder ist die hoffnung, dass der admin account immer der erste ist?

Reply:
glaube das wird bei mir lokal schon zu einem problem. erhalte

![Bildschirmfoto 2021-06-01 um 16 49 52](https://user-images.githubusercontent.com/1942953/120343995-77f32200-c2f9-11eb-82f4-cddb1d957409.png)

beim abfeuern der initial event records query.
applications in meiner datenbank:

![Bildschirmfoto 2021-06-01 um 16 51 18](https://user-images.githubusercontent.com/1942953/120344128-9822e100-c2f9-11eb-9a2b-b958bbdffbad.png)

Reply:
> welcher application ist hier first? is das egal? wird irgendeine verwendet, um ein token zu erstellen? oder ist die hoffnung, dass eine application eines admin accounts immer die erste ist?

nein hier geht es um die OAuth Applikationen eines Accounts.  Bei uns hat jeder Account/User eigentlich nur eine einzige OAuth Application.

Reply:
und die `resource` is der account? aah und am ende geht es eben nur, wenn man als jemand bestimmtes eingeloggt ist, um graphiql zu verwenden?
ich war ja im backend als admin eingeloggt und hab daraufhin ein 401 erhalten.

Reply:
ja genau, du kannst nur als angemeldeter User XY "deine" API testen.

Reply:
alles klar, konnte mich lokal mit "mobile-app@smart-village.app"-Account einloggen und erfolgreich eine Query abschicken.